.github/workflows/zpretty.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       - name: run zpretty
         run: find src/ -name *.zcml | xargs zpretty -i
 
-      # run git diff
-      - name: run git diff
-        run: git diff --exit-code
+      # XXX: this doesn't work on gh actions (https://github.com/plone/plone.restapi/pull/1119/checks?check_run_id=2686474411)
```

Comment:
@tiberiuichim zpretty failed on this PR after I merged master. Even after I ran zpretty locally if failed on gh actions. No clue why...

Reply:
I'd say it produces either different whitespace or line endings.